tests/it/main.rs

```diff
@@ -142,12 +153,15 @@ fn test_remove_all_optional() -> Result<()> {
     td.write(p, "test")?;
     assert!(td.remove_all_optional(p).unwrap());
 
-    // symlinks; broken and not
-    let p = Path::new("linksrc");
-    td.symlink("linkdest", p)?;
-    assert!(td.remove_all_optional(p).unwrap());
-    td.symlink("linkdest", p)?;
-    assert!(td.remove_all_optional(p).unwrap());
+    #[cfg(not(windows))]
```

Comment:
Kind of curious why you picked `not(windows)` vs `unix` here.

Reply:
Because `symlink` is also gated as `not(windows)`. But could be changed to `unix`. Not sure whether that would change Mac coverage.